### PR TITLE
Update eject to support all launchpad features

### PR DESCRIPTION
**Pull request by freiksenet**

### CODE REVIEW

File: frontend/services/eject/packageDotJson.js [hunk 1]
```diff
@@ -24,6 +24,6 @@ export default `{
     "body-parser": "^1.17.1",
     "express": "^4.15.2",
 {{dependencies}}
-    "graphql-server-express": "^0.6.0"
+    "graphql-server-express": "^1.2.0"
```
Comment (stubailo) [inline frontend/services/eject/packageDotJson.js]: This should be `apollo-server-express`

File: frontend/services/eject/serverDotJs.js [hunk 2]
```diff
@@ -7,18 +7,49 @@ export default `import express from 'express';
 import { graphqlExpress, graphiqlExpress } from 'graphql-server-express';
 import bodyParser from 'body-parser';
 
-import { schema, rootValue, context } from './schema';
+import * as Schema from './schema';
```
Comment (stubailo) [inline frontend/services/eject/serverDotJs.js]: Why this change?

Comment (freiksenet) [reply frontend/services/eject/serverDotJs.js]: Easier to copy logic from how the launchpad script works.

Comment (stubailo) [reply frontend/services/eject/serverDotJs.js]: Ah. I don't think this is what we should recommend for downloaded projects though - maybe we can change the launchpad script? I think we'd like the downloaded project to be a great starter kit for people.
